nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/resources/docs/org.apache.nifi.processors.slack.PostSlack/additionalDetails.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+    <head>
+        <meta charset="utf-8" />
+        <title>PostSlack</title>
+        <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+    </head>
+
+    <body>
+        <!-- Processor Documentation ================================================== -->
+        <h2>Description:</h2>
+        <p>
+            The PostSlack processor sends messages on <a href="https://www.slack.com">Slack</a>,
+            a team-oriented messaging service. The message can be a simple text message
+            or the FlowFile content (e.g. an image) can be uploaded and attached to the message too.
+        </p>
+        <p>
+            This processor uses <a href="https://api.slack.com/web">Slack Web API</a> methods
```

Comment:
Also we should just refer to the external documentation here instead of providing links, even though Slack's links are probably stable, IMO we should avoid the opportunity for dead links in our docs, as we'd need a new release to fix them.

Reply:
I found it in PutSlack, maybe it was not the best example.
I've checked the additionalDetails.html files of the processors and some of them use links too (eg. Consume/PublishAMQP, ExecuteFlumeSink/Source, PublishJMS).

Anyway, I'll remove the links if this is the best practice for the docs.

Reply:
Yeah I would've started with the existing processor too :) Might have been overlooked during that review, but I'd like this one to be really solid as it's a very helpful addition to the Slack bundle, so I want to make the user experience very easy with good documentation, so I think removing links kind of "future-proofs" the documentation. Thanks!